src/Scanner.java

```diff
@@ -1188,6 +1282,7 @@ private int predictSerializedSize() {
       size += start_key.length;  // The start key.
       size += 3;  // vint: stop key length (3 bytes => max length = 32768).
       size += stop_key.length;  // The stop key.
+      size += 1;  // bool: Whether the scanner is in reverse
```

Comment:
Nit: Don't need this hear since pre-protobuf versions don't support reverse scanning :)